templates/supertest/put/put.handlebars

```diff
@@ -1,4 +1,4 @@
-  it('should respond with {{length description}}', function(done) {
+  it('should respond with {{length description}}{{#if inputs}}, {{inputs.body.message}}{{/if}}', function(done) {
```

Comment:
the description is supplied by the Swagger spec....the specific response being tested, not part of the user-supplied data. This has been in since day one :) the issue was with its truncation, but i have a PR to this fork that allows truncation to be toggled off 

The `inputs.body.message` will have to be clarified by @mm-gmbd